Monika After Story/game/script-topics.rpy

```diff
@@ -4823,6 +4823,78 @@ label monika_coffee:
     m 1k "Maybe if I find out how, I'll tinker with the script so I can finally drink some! Ahaha~"
     return
 
+init 5 python:
+    for key in ['wolf','wolves','dog','dogs','domestication']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_domestication')
+    monika_random_topics.append('monika_domestication')
+    
+label monika_domestication:
+    m 3  "Do you ever think about wolves?"
+    m 3  "I guess a bit more specifically, how wolves turned into dogs."
+    m 3d "Like, don't you find it at least a little interesting how one of man's most fearsome enemies could turn into man's best friend?"
+    m 3d "I mean, with things like spiders and other creepy-crawlies, lots of people are still afraid of them despite never having even come across one that could hurt them in any way."
+    m 3d "Is it because we as a species learned to be afraid of the things that hurt us, thousands of years ago?"
+    m 3  "But whenever you see a picture of a wolf, I bet you just can't help but think 'Aww...'"
+    m 3g "Did humanity just have enough good experiences with dogs to override the bad experiences with wolves?"
+    return  
+     
+     
+init 5 python:
+    for key in ['olympics','athlete','2018','olympic']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_olympics')
+    monika_random_topics.append('monika_olympics')
+      
+label monika_olympics2018:
+    m 3  "You've heard about the Olympics happening in a few months, right?"
+    m 3a "Well, I was wondering if you'd maybe like to go to take me to watch the Olympics one year, if we're ever able to truly meet person-to-person."
+    menu:
+        m 5a "What do you think?"
+        
+        "Sure.":
+            m 5a "Really? I can't wait!"
+            m 5a "Although, I'm not sure how you'd compare to all those burly athletes, ehehe~"
+            m 5a "I'm only kidding~"
+            
+        "No thanks.":
+            m 1g "No? Alright..."
+            m 3k "There are still plenty more fun things for us to do together."
+            m 3a "If you ever decide you want to go somewhere or do something, don't hesitate to bring me along in a flashdrive. I love spending time with you~"
+            return
```

Comment:
`return` should only have one level of indentation.

Reply:
I think I've fixed all three errors. Please let me know if they're still incorrect, or if there's anything else I should do to make this better.